supported_os_controller.php

```diff
@@ -52,5 +56,217 @@ public function os()
         $obj->view('json', array('msg' => $out));
     }
     
+    /**
+     * Force data pull from supported_os GitHub
+     *
+     * @return void
+     * @author tuxudo
+     **/
+    public function update_cached_data()
+    {
+        // Authenticate
+        $obj = new View();
+        if (! $this->authorized()) {
```

Comment:
Please remove the authorised() block - not necessary anymore

Reply:
Removed